docko

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
```

Comment:
There is still quite a bit of functionality missing from this compared to the townies script (which is still included), but I think this format reads a bit better. no need to have every action in its own function.